### PR TITLE
vale

### PR DESCRIPTION
**Pull request by evgeni**
- **Vale**
- **testing**

* [x] I am okay with my commits getting squashed when you merge this PR.
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

File: .github/workflows/lint-with-vale.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  vale:
+    name: Linting with Vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
+        with:
+          filter_mode: diff_context
+          vale_flags: "--no-exit --minAlertLevel=error --glob=!*/doc-*/common/*"
```
Comment (apinnick) [inline .github/workflows/lint-with-vale.yml:14]: *.adoc will prevent the sym links problem

Comment (evgeni) [reply .github/workflows/lint-with-vale.yml:14]: you mean `--glob=*.adoc`?

Comment (apinnick) [reply .github/workflows/lint-with-vale.yml:14]: Yes

Comment (evgeni) [reply .github/workflows/lint-with-vale.yml:14]: Doesn't work for me locally.

Running it with `--glob='*.adoc'` (quoted, so my shell doesn't interpret the *) and it spins for minutes now.

Comment (apinnick) [reply .github/workflows/lint-with-vale.yml:14]: but now I'm  not certain. I have not used that flag.

File: .github/workflows/lint-with-vale.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+          # github-pr-annotations doesn't require elevated token privileges
+          reporter: github-pr-annotations
+          fail_on_error: true
```
Comment (ekohl) [inline .github/workflows/lint-with-vale.yml:17]: It has annotations, but the check doesn't appear to fail. That's confusing to me.

**Comment by ekohl** [comment]
https://github.com/theforeman/foreman-documentation/pull/3145 was merged.

**Pull request closed without merging**